### ISSUE TITLE
[BUG] Linux fails to compile wxTheApp not declared in this scope

### ISSUE THREAD

**Issue by noabody** [PCSX2/pcsx2#4233]
```
In file included from ../plugins/GSnull/GS.h:35,
                 from ../plugins/GSnull/Registers.cpp:19:
../common/include/PS2Eext.h: In function ‘void SysMessage(const char*, ...)’:
../common/include/PS2Eext.h:238:3: error: ‘wxTheApp’ was not declared in this scope
  238 |   wxTheApp->CallAfter([&] {
      |   ^~~~~~~~
```
It's just missing an include for the ```__unix__``` platform.  This patch resolves the compilation issue.
```diff
diff --git a/common/include/PS2Eext.h b/common/include/PS2Eext.h
index 4f4bbbcc1..dfa854c1e 100644
--- a/common/include/PS2Eext.h
+++ b/common/include/PS2Eext.h
@@ -37,6 +37,7 @@
 
 #include <cstring>
 #include <wx/msgdlg.h>
+#include <wx/app.h>
 
 #define EXPORT_C_(type) extern "C" __attribute__((stdcall, externally_visible, visibility("default"))) type
 
```

**Comment by noabody** [comment]
My build platform 64-bit Manjaro Linux 20.2.1, GCC 10.2.0.  I see there is a note about clang format on commit [e883e93](https://github.com/PCSX2/pcsx2/commit/e883e93f526d2b3df07fc41cf193be4610b80497) but fail to see how that would affect a missing include.

**Comment by refractionpcsx2** [comment]
There is already a PR to fix this #4228

**Comment by refractionpcsx2** [comment]
Ok it's now merged.
